packages/projection/src/sinks/inMemory/types.ts

```diff
@@ -10,6 +10,7 @@ export type InMemoryStore = {
       retirements: PoolRetirement[];
     }
   >;
+  adaHandles: Map<Cardano.Address, string>;
```

Comment:
In order to implement RollBackward in `sink()` we need to store history of handle owners (e.g. an array): rolling back a transaction that transfers ownership of a handle should result in re-assigning that handle to the previous owner. Also, shouldn't it be a mapping from handle to address (not the other way around)?
```suggestion
  // type AddressAlias = string;
  // type AddressAliasProtocol = string;
  addressAliases: Map<AddressAlias, Map<AddressAliasProtocol, Cardano.Address[]>>;
```